src/CheckoutSdk/Sessions/SessionRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Checkout.Common;
+using Checkout.Sessions.Channel;
+using Checkout.Sessions.Completion;
+using Checkout.Sessions.Source;
+
+namespace Checkout.Sessions
+{
+    public class SessionRequest
+    {
+        public SessionSource Source { get; set; }
+        public long? Amount { get; set; }
+        public Currency Currency { get; set; }
```

Comment:
The property Currency could be nullable? : public Currency? Currency { get; set; }

Reply:
Nice! I have just added this changes. Thanks